CONTRIBUTING.md

```diff
@@ -85,6 +85,36 @@ This will setup your project along with firebase emulator in a docker environmen
 
 You should fill in these values in their relevant fields in the `.env` file.
 
+
+### Firebase Emulator Setup
+
+From your parent directly run cd/functions
+1. run 
```

Comment:
I believe this step is not required, can you re-check if this is mandatory. I dont remember running this while setting up in my local machine.